### PR TITLE
chore: update typography examples

### PR DESCRIPTION
**Pull request by hzhu**
## Description

When building out the Typography component page @allisonacs noticed that some examples (Lists and Code highlight) [weren't clear](https://github.com/zendeskgarden/website/pull/100#pullrequestreview-446058820). 

## Detail

This PR updates the copy and adds examples which clearly demonstrate the following:

1) Types of lists, e.g. ordered vs unordered
2) List sizes
3) Code highlight background colors
4) Code highlight sizes

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/examples/typography/ListSize.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { UnorderedList } from '@zendeskgarden/react-typography';
+
+const StyledDiv = styled.div`
+  margin-bottom: 18px;
```
[the remark this answers is not in the document]
Comment (jzempel) [reply src/examples/typography/ListSize.tsx:13]: Ditto for `Lists.tsx`.

File: src/pages/components/typography.mdx [hunk 2]
```diff
@@ -23,14 +23,22 @@ components:
...
-import Code from '../../examples/typography/Code.tsx';
-import CodeCode from '!!raw-loader!../../examples/typography/Code.tsx';
+import CodeSize from '../../examples/typography/CodeSize.tsx';
+import CodeSizeCode from '!!raw-loader!../../examples/typography/CodeSize.tsx';
 
-<CodeExample code={CodeCode}>
-  <Code />
+<CodeExample code={CodeSizeCode}>
+  <CodeSize />
+</CodeExample>
```
Comment (jzempel) [inline src/pages/components/typography.mdx:33]: Clicking "View code" on this example results in a hard site crash. See https://5f0ce5cbe13983e8523fa327--zendeskgarden.netlify.app/components/typography#lists for details.

File: src/examples/typography/CodeSize.tsx [hunk 3]
```diff
@@ -10,20 +10,22 @@ import styled from 'styled-components';
...
+      </Code>
     </StyledDiv>
     <StyledDiv>
-      <Code size="medium" hue="green">
+      <Code hue="grey" size="medium">
         Veggies es bonus vobis
       </Code>
     </StyledDiv>
-    <Code size="large" hue="yellow">
+    <Code hue="grey" size="large">
```
Comment (jzempel) [inline src/examples/typography/CodeSize.tsx:28]: Don't add the default `hue="grey"` to this example. It was only needed in order to represent specific treatment in the color-based example.

Comment (hzhu) [reply src/examples/typography/CodeSize.tsx]: I've removed `hue=grey` here and added it to the color examples. Apologies for the confusion, the Google Doc example threw me off as it only showed three colors and I hadn't checked hues that were allowed.

**Pull request merged**